BCnEnc.Net/BCnEncoder.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>netcoreapp31</TargetFramework>
```

Comment:
Was this changed on purpose?

Reply:
Ah, this is an error on my part. I tested the decoding capability yesterday, while encoding was still missing, so I used the local code and I had to switch to netcoreapp31. I will revert that back to netstandard2.1